### DISCUSSION TITLE
Sockets to use WebRTC

### DISCUSSION THREAD

**Discussion by kevinmamaqi** [supabase/supabase#1301] [Questions] [answered]
Are there examples with webRTC and WebSockets on supabase?

**Comment by mjyoung** [comment]
I'm also interested in if this is possible with Supabase for peer to peer video using websockets and webrtc

**Comment by w3b6x9** [comment, marked as answer]
We announced the new Realtime Broadcast (PubSub over Websockets) so you can use that for WebRTC signaling once it's launched to the general public.

**Reply by swk23C8** [reply]
> 

Just checked out the multiplayer demo! Can't wait to use it

**Reply by naonvl** [reply]
I'll need webrtc + mediasoup for audio/video call purposes. Does the implementation still the same or identical on edge functions?

**Reply by danicunhac** [reply]
@w3b6x9 do you have an example for WebRTC signaling? I'm building a live video call and trying with Supabase to manage everything

**Reply by jtkeyva** [reply]
@w3b6x9 any updates with examples? does this eliminate the need for turn or stun servers?

**Reply by jadenmazzone** [reply]
I am also looking for an update on this, did anyone figure this out?
